### PR TITLE
chore(spec): Ensure /var/lib/%{name} is created

### PR DESCRIPTION
**Pull request by m-horky**
The crypto module tries to create its temporary directories in /var/lib/ when running as root. This patch creates that directory.

### CODE REVIEW

File: insights-ansible-playbook-verifier.spec [hunk 1]
```diff
@@ -28,13 +28,15 @@ playbooks generated by Red Hat Insights.
...
 mv %{buildroot}%{_bindir}/%{name} %{buildroot}%{_libexecdir}
+mkdir -p %{buildroot}/%{_sharedstatedir}/%{name}/
 %pyproject_save_files insights_ansible_playbook_lib insights_ansible_playbook_verifier
 
 %check
 %pyproject_check_import insights_ansible_playbook_lib insights_ansible_playbook_verifier
 
 %files -n %{name} -f %{pyproject_files}
 %{_libexecdir}/%{name}
+%attr(750,root,root) %dir %{_sharedstatedir}/%{name}/
```
Comment (ptoscano) [inline insights-ansible-playbook-verifier.spec:39]: s/750/700/

File: insights-ansible-playbook-verifier.spec [hunk 2]
```diff
@@ -28,13 +28,15 @@ playbooks generated by Red Hat Insights.
 %pyproject_install insights_ansible_playbook_lib insights_ansible_playbook_verifier
 mkdir -p %{buildroot}/%{_libexecdir}
 mv %{buildroot}%{_bindir}/%{name} %{buildroot}%{_libexecdir}
+mkdir -p %{buildroot}/%{_sharedstatedir}/%{name}/
```
Comment (ptoscano) [inline insights-ansible-playbook-verifier.spec:31]:
```
$ grep -rn _sharedstatedir /usr/lib/rpm/macros*
%_sharedstatedir        %{_prefix}/com
```
definitely not the right macro to use...

rather use:
```suggestion
mkdir -p %{buildroot}%{_localstatedir}/lib/%{name}
```

File: insights-ansible-playbook-verifier.spec (hunk 1 above)
Comment (ptoscano) [inline insights-ansible-playbook-verifier.spec:39]: ditto (`%{_sharedstatedir}`)

**Review by ptoscano — CHANGES_REQUESTED** [review]

File: insights-ansible-playbook-verifier.spec (hunk 2 above)
Comment (m-horky) [reply insights-ansible-playbook-verifier.spec:31]: Interesting, https://docs.fedoraproject.org/en-US/packaging-guidelines/RPMMacros/ defines it as /var/lib.

**Review by ptoscano — APPROVED** [review]

**Pull request merged**